src/schema/core.yaml

```diff
@@ -471,6 +472,13 @@ slots:
     domain: person value
     range: string
 
+  email:
+    description: >-
+      An email address for an entitity such as a person
+    range: string
+    notes: >-
+      More specific domain? Acceptable pattern? https://emailregex.com/ ?
```

Comment:
Keep this kind of discussion in the ticket/PR, keep the schema as clean as possible.

This seems over the top:
https://emailregex.com/

Reply:
@cmungall I just merged this. Do you me to remove the notes and do another PR?